lib/element.rb

```diff
@@ -202,16 +238,6 @@ def text
     element.text
   end
 
-  def text=(text)
-    element.clear
-    element.send_keys(text)
-  end
-
-  def value
```

Comment:
this method needs to remain as well for similar reasons. I see this was moved to a different line.